cypress/e2e/ProductionSmokeTests/$SBIR/productionTests.js

```diff
@@ -39,4 +39,49 @@ And('page title is {string}', (title) => {
 
 And('the text {string} is displayed', (str) => {
     cy.get(`p:contains("${str}")`).should('be.visible');
+});
+
+Then('all displayed events have a future date', () => {
+    const months = [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December'
+    ];
+    const date = new Date();
+    const year = date.getFullYear();
+    const month = date.getMonth();
+    const day = date.getDate();
+
+
+    cy.get('div.event-date-time').each(($el) => {
+        const monthAndDay = ($el[0].innerText.split(','))[0];
+        const eventMonth = monthAndDay.split(' ')[0];
+        const eventDay = monthAndDay.split(' ')[1];
+        const eventYear = ($el[0].innerText.split(','))[1].trim();
```

Comment:
There are calls to `parseInt` every time these variables are used. You should call it as part of assigning these values in the first place and not need to think about it further down.

Reply:
Yep, fixed!